app/controllers/comments_controller.rb

```diff
@@ -22,10 +22,11 @@ def edit
   # POST /comments or /comments.json
   def create
     @comment = Comment.new(comment_params)
+    @comment.author = current_user
 
     respond_to do |format|
       if @comment.save
-        format.html { redirect_to comment_url(@comment), notice: "Comment was successfully created." }
+        format.html { redirect_to fallback_location: root_path, notice: "Comment was successfully created." }
```

Comment:
Why list no url while listing a fallback_location? Is this meant to be a redirect_back?

Reply:
fixed